docker/entrypoint.sh

```diff
@@ -1,6 +1,26 @@
 #!/bin/bash
 
-kolibri manage migrate
-kolibri services --background
+if [ $RUN_BG_SERVICE = 'true' ]
+then
+    kolibri services --background
+fi
+
+# plugin enable
+kolibri plugin enable kolibri.plugins.app
+kolibri plugin disable kolibri.plugins.learn
+
+# Disable and enable the kolibri_explore_plugin to ensure that the static files
+# are correctly copied to the static folder
+kolibri plugin disable kolibri_explore_plugin
+kolibri plugin enable kolibri_explore_plugin
+
+# Initial populate
+if [ $POPULATE_CONTENT = 'true' -a ! -e $KOLIBRI_HOME/populated ]
```

Comment:
Nit: why not just `if [ ! -e $KOLIBRI_HOME/populated ]`. In other words, why is `POPULATE_CONTENT` useful at all if it's always true?